### DISCUSSION TITLE
Miniman YAML config for standalone binary

### DISCUSSION THREAD

**Discussion by Zabrane** [goreleaser/nfpm#336] [Q&A] [answered]
Hi guys

I'm trying to ship a standalone (no dependencies) binary for `Ubuntu 20.04 LTS` on my Mac.

```
$ brew install goreleaser/tap/nfpm
$ nfpm --version                                                                                                                                                                         
nfpm version nfpm version 2.5.1
commit: 5c1976b7cd61db8d9d5a5165de09695d1c159748
built at: 2021-05-03T13:26:00Z
built by: goreleaser
module version: v2.5.1, checksum: h1:xZpdwr/wgp2OmfxjqJcw5CeHHnrVvz75ZRzCpSXeSNo=

$ cat nfpm.yaml
name: "crypto-cli"
arch: "amd64"
platform: "linux"
version: "v1.0.0"
section: "default"
priority: "extra"
replaces:
- crypto-cli
maintainer: "xxxxxxx"
description: |
  Crypto Client.
vendor: "xxxxxx"
homepage: "https://www.xxxxx.com/"
license: "Free Software"
contents:
- src: crypto-cli
  dst: /usr/local/bin/crypto-cli
#overrides:
#  deb:
#    scripts:
#      postinstall: ./scripts/postinstall.sh
#      preremove: ./scripts/preremove.sh




$ nfpm pkg --packager deb --target .
$ ls -1 *.deb
crypto-cli_1.0.0_amd64.deb
```

All went well but I'm not sure what to do with the `overrides` section.
Is it OK to not provide these pre/post scripts? If Yes, will my package be automatically updated when i will distribute version `2.0.0`. Example version `1.0.0` is installed on some server, then the admin runs:
```
# apt update
# apt upgrade
```

will my package gets automatically updated to `2.0.0`.

Finally, is there anything specific i need to be aware of to make my package works pefectly on `Ubuntu 20.04 LTS` (my only target OS for now)?

Thanks for this fantastic tool.

**Comment by djgilcrease** [comment, marked as answer]
The post install and pre remove scripts are not needed unless you need to do some extra setup after install or some extra cleanup after remove. 

Nothing specific you need to do to make it work so long as the binary will run.
